src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -208,74 +241,20 @@ public String toUrlValue() {
     }
     return addressType;
   }
+    
+  /**
+   * Enum lookup based search string.
+   * @param addressType finding addressType string.
+   * @return specified AddressType enum value or if it's not found then {@value AddressType#UNKNOWN}.
+   */
+  public static AddressType lookup(final String addressType) {
+      AddressType result = VALUE_MAP.get(addressType.toLowerCase(Locale.ENGLISH));
+      if (null != result) {
+          return result;
+      }
 
-  public static AddressType lookup(String addressType) {
-    if (addressType.equalsIgnoreCase(STREET_ADDRESS.toString())) {
```

Comment:
I think this class was missed in change e2aace0c337. Should use the same approach, I think.